src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -50,9 +50,20 @@ public CommandResult execute() {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
-                matchedPersons.add(person);
+
+            //case-insensitive FindCommand
+            for(String nameWords : wordsInName){
+                for(String keyword : keywords){
+                    if(nameWords.equalsIgnoreCase(keyword)){
+                        matchedPersons.add(person);
+                    }
+                }
             }
+
+            //case sensitive FindCommand
+            //if (!Collections.disjoint(wordsInName, keywords)) {
+            //    matchedPersons.add(person);
+            //}
```

Comment:
Remember to remove the code you are no longer using. However, since you want to implement some sort of a toggle between case-sensitivity and case-insensitivity, I guess it is fine.